test/unit/StakingPool/helpers.js

```diff
@@ -29,6 +29,61 @@ const setTime = async timestamp => {
   await mineNextBlock();
 };
 
+async function calculateBasePrice(stakingPool, product, priceChangePerDay) {
```

Comment:
Try making the functions as stateless as possible - i.e. avoid making them dependant on the current time or other state. Instead the needed state as arguments.
- It is not clear from the title that the function depends on the current time. If I was working on a test that uses this function I might move it within the test and not be aware that it should be called at a particular time.
- The `stakingPool` parameter is not used at all.